react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+export default function Stopwatch() {
+  let [time, setTime] = useState(0);
+  let [running, setRunning] = useState(false);
```

Comment:
This is going to hold the return from `setInterval`, so it's best to use an initial value of `undefined`. (Since `undefined` is the default, you can just call `useState();`.)